pkg/fw/Dockerfile

```diff
@@ -54,6 +54,11 @@ ADD ${RPI_BT_FIRMWARE_URL}/${RPI_BT_FIRMWARE_VERSION}/broadcom/BCM4345C0.hcd .
 ADD ${RPI_BT_FIRMWARE_URL}/${RPI_BT_FIRMWARE_VERSION}/broadcom/BCM43430B0.hcd .
 ADD ${RPI_BT_FIRMWARE_URL}/${RPI_BT_FIRMWARE_VERSION}/broadcom/BCM4345C5.hcd .
 
+# Hailo 8 GPU firmware
+ENV HAILO_FW_VERSION 4.16.0
+RUN mkdir -p /lib/firmware/hailo
```

Comment:
The destination directory (along with its child dirs) will be created by ADD, so you don't need this RUN here...

Reply:
fixed.